packages/core/src/Mode.ts

```diff
@@ -18,8 +18,9 @@ export interface ModeDefinition {
 export enum RuleProperty {
     EDITABLE = 'editable',
     BREAKABLE = 'breakable',
+    RENDERBR = 'renderbr',
```

Comment:
How about `ALLOW_EMPTY` ?

Reply:
Although isn't it a little weird (although it sure _is_ convenient) to use `Mode` to configure something that is so tied to the DOM rendering? It seems like it should rather be done in the configuration of the plugin that declares the DOM renderer.

Reply:
+1 for `ALLOW_EMPTY`

I agree that the property is not likely to change between modes.
And indeed, the cascading expressivity that the ContextManager provide is nice.

We could re-use the ContextManager by creating a different logic for it, but I don't think it is necessary.

With that in mind, I believe we can think of the mode as a CSS sheet. The sheet can change or not but is a way to configure properties in a cascading fashion.

And the properties that can be configured can be any property that a plugin might need. Even a rendering plugin.
In the future, we could have differents rendering depending on the mode for a particular node.

Reply:
I validate ALLOW_EMPTY. I suggest to keep the feature in Mode for now, even though I agree that it is a bit weird.

Reply:
`ALLOW_EMPTY = 'allow_empty',` ?

Reply:
I think `ALLOW_EMPTY = 'allowEmpty',` would be better considering @Zinston coded the `is` in such a way that, when the mode gives no information about a particular property, it looks for it on the node *in case* the node itself might define it. So since it *might* be possible to set this property on a `VNode` (though I don't see why we would), it makes sense to keep camelCasing it.